SnaffCore/ActiveDirectory/AdData.cs

```diff
@@ -60,40 +78,38 @@ public AdData()
             {
                 Mq.Trace("Target DC and Domain specified: " + MyOptions.TargetDomain + " + " + MyOptions.TargetDc);
                 _targetDc = MyOptions.TargetDc;
-                _targetDomain = MyOptions.TargetDomain;
+                _targetDomain = Domain.GetDomain(new DirectoryContext(DirectoryContextType.Domain, MyOptions.TargetDomain));
```

Comment:
these need to just be strings taken straight from whatever the user specified, to avoid those Domain.GetDomain calls.